WordPress/src/main/java/org/wordpress/android/ui/reader/ReaderPostListFragment.java

```diff
@@ -654,6 +654,9 @@ public void onLoadData() {
                     // make sure swipe-to-refresh progress shows since this is a manual refresh
                     mRecyclerView.setRefreshing(true);
                 }
+
+                ReaderPostTable.purgeUnbookmarkedPostsWithBookmarkTag();
```

Comment:
We might want to consider wrapping these two statements into 
` if (getCurrentTag() != null && getCurrentTag().isBookmarked())` so we don't force-refresh the post list for all the tags. Wdyt?

Reply:
Yeah, I Agree! It makes much more sense in this way you're suggesting.